packages/esm-service-queues-app/translations/en.json

```diff
@@ -196,7 +197,6 @@
   "queueLocation": "Queue Location",
   "queueLocationRequired": "Queue location is required",
   "queueName": "Queue name",
-  "queueNumber": "Queue Number",
```

Comment:
Why is this getting removed?

Reply:
Turns out I forgot to have a case statement for this column. Thanks for catching.